### PR TITLE
Settings Card Component

### PR DESCRIPTION
**Pull request by eddieferrer**
* Refactor card used on settings pages into a new component
* Add component story
* Remove duplicate CSS and component imports

### CODE REVIEW

File: src/components/Kv/KvSettingsCard.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.settings-card {
+	background: $white;
+	padding: 1.95rem;
+	display: grid;
```
Comment (ryan-ludwig) [inline src/components/Kv/KvSettingsCard.vue:33]: Does this component still render semi-ok in IE?

Comment (eddieferrer) [reply src/components/Kv/KvSettingsCard.vue:33]: @ryan-ludwig It doesnt look too bad without the grid support. 

Screen Shot 2020-08-06 at 4 12 15 PM

Comment (ryan-ludwig) [reply src/components/Kv/KvSettingsCard.vue:33]: Yeah, not bad at all, nice!

Comment (eddieferrer) [reply src/components/Kv/KvSettingsCard.vue]: @ryan-ludwig I decided to refactor this to use flex instead for better IE support. There really was no reason to use grid here.

**Pull request merged**